friends-in-the-nineteenth-century-volume-1/modernized/08-chapter-7.adoc

```diff
@@ -819,17 +855,24 @@ J+++.+++ J. Gurney had by this time manifested,
 in his numerous publications (irrespective of the last named Brief Remarks on Interpretation),
 the following points of variance from the settled characteristic doctrines of Friends.
 
+[.embedded-content-document]
+--
+
+[.discourse-part]
```

Comment:
I'm not sure if this is right, but this whole part seemed to need more separation than  regular paragraphs. I looked in the style guide, but couldn't find anything that seemed to suit it. I may have missed it!

Reply:
yeah, there's no perfect syntax for a list like this. I was thinking that maybe using the syllogism syntax might look the best, since it makes a bullet point list, which seems kind of fitting here. Let's try that.